### PR TITLE
drivers: wifi: twt tx fix

### PR DESCRIPTION
**Pull request by ajayparida**
[SHEL-1179] Fix for twt tx issue.

Signed-off-by: Ajay Parida <ajay.parida@nordicsemi.no>

### CODE REVIEW

File: drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h [hunk 1]
```diff
@@ -263,7 +263,6 @@ struct wifi_nrf_fmac_buf_map_info {
 	unsigned long nwb;
 };
 
-
```
Comment (krish2718) [inline drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h:266]: unnecessary whitespace

File: drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c [hunk 2]
```diff
@@ -229,27 +245,16 @@ void wifi_nrf_event_proc_twt_sleep_zep(void *vif_ctx,
...
-				pkt_info =
-				&fmac_dev_ctx->tx_config.pkt_info_p[desc];
-
-				txq = pkt_info->pkt;
-
-				tx_cmd_init(fmac_dev_ctx,
-					    txq,
-					    desc,
-					    pkt_info->peer_id);
+		/*for (ac = WIFI_NRF_FMAC_AC_VO; ac >= 0; --ac) {*/
```
Comment (krish2718) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_twt.c:250]: Remove commented code

**Review by krish2718 — CHANGES_REQUESTED** [review]
Can you please add the fix details to the commit log? It is not clear as to what we are trying to fix.
Also, please avoid unnecessary whitespace changes?

File: drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h [hunk 3]
```diff
@@ -283,6 +283,8 @@ struct rpu_host_stats {
 	unsigned long long total_tx_pkts;
 	unsigned long long total_tx_done_pkts;
 	unsigned long long total_rx_pkts;
+	int tx_cmd_cnt;
```
Comment (krish2718) [inline drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h:286]: This can be a separate commit

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

File: drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c [hunk 4]
```diff
@@ -91,27 +91,29 @@ void tx_desc_free(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
...
+			case 1:
+				fmac_dev_ctx->tx_config.spare_desc_queue_map &= 0xff0f;
+				break;
+			case 2:
+				fmac_dev_ctx->tx_config.spare_desc_queue_map &= 0xf0ff;
+				break;
+			case 3:
+				fmac_dev_ctx->tx_config.spare_desc_queue_map &= 0x0fff;
+				break;
+			}
```
[the remark this answers is not in the document]
Comment (ajayparida) [reply drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c:114]: yes, it is aligned as per the statement

**Pull request closed without merging**